test/deploy/60_ocmagent.ManagedFleetNotificationRecord.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: ocmagent.managed.openshift.io/v1alpha1
+kind: ManagedFleetNotificationRecord
+metadata:
+  name: test-fleet-notification-record
```

Comment:
I understand this is test yaml but between multiple ManagedFleetNotificationRecord resources, will need to uniquely identify the resource specific to a MC cluster for which the record is created. What about having this name as name or cluster ID for MC cluster and removing `.status.managementCluster` (would be duplicate info if `.metadata.name` already has it.